frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/compat/kotlinUIDslBuilderCompat.kt

```diff
@@ -111,3 +118,21 @@ fun Panel.rowCompat(init: Row.() -> Unit): Row {
   )
   return result as Row
 }
+
+fun GitBranchCheckoutOperation.performCompat(startPoint: String, options: GitNewBranchOptions) {
+  val method = gitBranchCheckoutOperationImpls.getDeclaredMethod(
+    "perform\$default",
```

Comment:
Meh, this probably won't work... TBD if such method even existed before this very EAP

Reply:
For the record, `git4idea.ui.branch.GitBranchCheckoutOperation` class in the previous EAP:

```java
// Compiled from "GitBranchCheckoutOperation.kt"
public final class git4idea.ui.branch.GitBranchCheckoutOperation {
  public static final git4idea.ui.branch.GitBranchCheckoutOperation$Companion Companion;
  public git4idea.ui.branch.GitBranchCheckoutOperation(com.intellij.openapi.project.Project, java.util.List<? extends git4idea.repo.GitRepository>);
  public final void perform(java.lang.String, git4idea.branch.GitNewBranchOptions);
  public static final com.intellij.openapi.diagnostic.Logger access$getLOG$cp();
  static {};
}
```

and in this EAP:

```java
// Compiled from "GitBranchCheckoutOperation.kt"
public final class git4idea.ui.branch.GitBranchCheckoutOperation {
  public static final git4idea.ui.branch.GitBranchCheckoutOperation$Companion Companion;
  public git4idea.ui.branch.GitBranchCheckoutOperation(com.intellij.openapi.project.Project, java.util.List<? extends git4idea.repo.GitRepository>);
  public final void perform(java.lang.String, git4idea.branch.GitNewBranchOptions, java.lang.Runnable);
  public static void perform$default(git4idea.ui.branch.GitBranchCheckoutOperation, java.lang.String, git4idea.branch.GitNewBranchOptions, java.lang.Runnable, int, java.lang.Object);
  static void performCheckout$default(git4idea.ui.branch.GitBranchCheckoutOperation, java.lang.String, java.lang.String, boolean, boolean, java.lang.Runnable, int, java.lang.Object);
  static void checkout$default(git4idea.ui.branch.GitBranchCheckoutOperation, java.lang.String, java.lang.String, git4idea.ui.branch.GitBranchCheckoutOperation$Companion$CheckoutConflictResolution, java.lang.Runnable, int, java.lang.Object);
  public static final com.intellij.openapi.diagnostic.Logger access$getLOG$cp();
  static {};
}
```